src/WizardLayout/WizardLayout.tsx

```diff
@@ -110,11 +110,11 @@ export default class WizardLayout extends React.PureComponent {
           </FlexBox>
         </FlexBox>
         <FlexBox className={cssClass.FOOTER}>
-          <Button
+          { onSaveAndExit && <Button
```

Comment:
Nit: This is subtle, and I see that this file has done it inconsistently, but there should be no spacing inside JSX brackets, and there should be inside of JS object brackets.

So in this example it would look like:
```jsx
{onSaveAndExit && <Button ... />}
```

Sometimes you get object brackets inside of JSX brackets which is when you get the funny thing like:
```jsx
<Select
  value={{ label: "label", value: "value" }}
  ...
/>
```

Just a heads up so you can be the change whenever you see this done incorrectly around our codebases!